src/network/networkctl.c

```diff
@@ -2507,6 +2507,7 @@ typedef struct LLDPUserdata {
         Table *table;
         int *neighbors_count;
         uint16_t *capabilities_all;
+        JsonVariant **json_list;
```

Comment:
hmm, why not just make this a proper member of the structured that is freed along with it? i.e. this indirection via 2nd level pointer is unnecessary. there's no benefit of maintaining the pointer to the json variant object on the stack rather than right here inside this structure that itself is on the stack anyway